addons/auth_passkey/views/auth_passkey_login_templates.xml

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <template priority="32" id="auth_passkey_login" inherit_id="web.login">
-        <xpath expr="//div[hasclass('oe_login_buttons')]" position="after">
-            <em t-attf-class="d-block text-center text-muted small my-1">- or -</em>
-            <div class="text-center mt-2">
-                <button class="btn btn-link btn-sm passkey_login_link p-0 border-0" type="button">Log in with Passkey</button>
-            </div>
-        </xpath>
         <xpath expr="//input[@name='redirect']" position="before">
             <input type="hidden" name="webauthn_response"/>
         </xpath>
     </template>
+
+    <template id="auth_passkey_login_oauth" inherit_id="web.login_oauth" name="Passkey Login OAuth">
+        <xpath expr="//div" position="before">
```

Comment:
Instead of targeting a random div and adding it before, target the t-set (`//t[@t-set='auth_btns']`) and put it `after`: this will be way more robust to future refactoring and custo made by the ps tech.